diff_cover/violationsreporters/java_violations_reporter.py

```diff
@@ -115,3 +116,46 @@ def installed(self):
         Returns: boolean False: As findbugs analyses bytecode, it would be hard to run it from outside the build framework.
         """
         return False
+
+
+class PmdXmlDriver(QualityDriver):
+    def __init__(self):
+        """
+        See super for args
+        """
+        super(PmdXmlDriver, self).__init__(
+            'pmd',
+            ['java'],
+            []
+        )
+
+    def parse_reports(self, reports):
+        """
+        Args:
+            reports: list[str] - output from the report
+        Return:
+            A dict[Str:Violation]
+            Violation is a simple named tuple Defined above
+        """
+        violations_dict = defaultdict(list)
+        for report in reports:
+            xml_document = cElementTree.fromstring("".join(report))
+            files = xml_document.findall(".//file")
+            for file in files:
+                for error in file.findall('violation'):
+                    line_number = error.get('beginline')
+                    error_str = "{}: {}".format(error.get('rule'),
+                                                error.text.strip())
+                    violation = Violation(int(line_number), error_str)
+                    filename = GitPathTool.relative_path(file.get('name'))
+                    filename = filename.replace(os.sep, "/")
+                    violations_dict[filename].append(violation)
+
+        return violations_dict
+
+    def installed(self):
+        """
+        Method checks if the provided tool is installed.
+        Returns: boolean False: As findbugs analyses bytecode, it would be hard to run it from outside the build framework.
+        """
+        return True
```

Comment:
I think you want `return False` here. 

Diff-cover is unlikely to be running findbugs, instead it will be parsing a report provided by the user.

If you intend to have diff-cover run findbugs this PR is going to need a bit more work to actually be able to run it.